frontend/src/pages/BoardPage/components/BoardCategory/index.styles.ts

```diff
@@ -63,3 +70,32 @@ export const DropdownIcon = styled(ChevronDown)`
 export const OptionList = styled(Dropdown.OptionList)`
   left: 46%;
 `;
+
+export const DesktopCategoryContainer = styled.div`
+  width: 100%;
+  display: flex;
+  overflow: auto;
+  border-bottom: 1px solid ${props => props.theme.colors.gray_400};
+  gap: 20px;
+  margin-top: 2em;
+  @media (max-width: 875px) {
+    display: none;
+  }
+`;
+
+export const DesktopCategoryStyle = css`
+  font-family: 'BMHANNAAir', 'Noto Sans KR';
+  font-size: 1.5rem;
```

Comment:
```suggestion
  font-size: 1.3rem;
```

로고 `속닥속닥`보다 글씨 크기가 커서 조금 부자연스럽네요!
`1.3rem` 정도면 적당할 것 같은데 어떠신가요!